spec.html

```diff
@@ -32143,6 +32143,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are described using <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons.
+          Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by the [[Identifier]] property of the records returned by AvailableTimeZoneIdentifiers.
+          They may also be <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> represents the preferred identifier of a named time zone, while a <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an alternate identifier for the same named time zone.
+          The set of built-in, named time zone identifiers (both primary and non-primary) that are supported by an ECMAScript implementation are called available time zone identifiers.
+        </p>
+        <p>
+          At a minimum, ECMAScript implementations must support a built-in time zone with the identifier *"UTC"*, which must be primary.
+          If the return value _tz_ of SystemTimeZoneIdentifier is different from *"UTC"*, and IsTimeZoneOffsetString(_tz_) is *false*, then implementations must also support _tz_ as a built-in time zone identifier, and it must be primary as well.
+          In addition, implementations may support any number of other built-in time zones.
+        </p>
+        <p>
+          Implementations that follow the requirements for time zones as described in the ECMA-402 Internationalization API specification are called <dfn>time zone aware</dfn>.
+          Time zone aware implementations must define built-in named time zones corresponding to the Zone and Link names of the IANA Time Zone Database (and <strong>only</strong> such names) and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations as specified in the ECMA-402 specification.
```

Comment:
```suggestion
          Time zone aware implementations must define built-in named time zones corresponding to the Zone and Link names of the IANA Time Zone Database (and <b>only</b> such names) and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations as specified in the ECMA-402 specification.
```

Reply:
I removed this after you reviewed it last time, and it crept back in because I copy/pasted it from the Temporal spec again. I just fixed it on the Temporal side too so it shouldn't come back again. :-)

My latest commit fixes this and your other suggestions. Thanks for the review!